src/rsa/bigint.rs

```diff
@@ -132,6 +132,18 @@ impl Positive {
         let bits = unsafe { GFp_BN_num_bits(self.as_ref()) };
         bits::BitLength::from_usize_bits(bits)
     }
+
+    // TODO Does this need to be constant time?
+    pub fn subtract_smaller(&self, smaller: &Positive) -> Result<Positive, error::Unspecified> {
```

Comment:
We should define all the math in terms of `Elem`s, in particular using only modular arithmetic. In this case, we can use `elem_sub`. (We want to use only modular arithmetic because we want to minimize code size and code complexity.)

Eventually we'll change `elem_sub` and all the other modular arithmetic is constant-time. The existing use of `elem_sub` needs it to be constant-time too. But it isn't a priority now and the plan for doing so first requires us to simplify the `BIGNUM` code more. I'm working on it as I have time now.